src/Raven.Studio/typescript/models/database/tasks/periodicBackup/amazonSettings.ts

```diff
@@ -12,27 +12,31 @@ abstract class amazonSettings extends backupSettings {
     allowedRegions: Array<string>;
 
     static availableAwsRegionEndpointsStatic = [
+        { label: "Africa (Cape Town)", value: "af-south-1", hasS3: true, hasGlacier: true },
+        { label: "Asia Pacific (Hong Kong)", value: "ap-east-1", hasS3: true, hasGlacier: true },
         { label: "Asia Pacific (Mumbai)", value: "ap-south-1", hasS3: true, hasGlacier: true },
-        { label: "Asia Pacific (Seoul)", value: "ap-northeast-2", hasS3: true, hasGlacier: true },
         { label: "Asia Pacific (Osaka-Local)", value: "ap-northeast-3", hasS3: true, hasGlacier: true },
+        { label: "Asia Pacific (Seoul)", value: "ap-northeast-2", hasS3: true, hasGlacier: true },
         { label: "Asia Pacific (Singapore)", value: "ap-southeast-1", hasS3: true, hasGlacier: true },
         { label: "Asia Pacific (Sydney)", value: "ap-southeast-2", hasS3: true, hasGlacier: true },
         { label: "Asia Pacific (Tokyo)", value: "ap-northeast-1", hasS3: true, hasGlacier: true },
-        { label: "AWS GovCloud (US)", value: "us-gov-west-1", hasS3: true, hasGlacier: true },
-        { label: "AWS GovCloud (US)", value: "fips-us-gov-west-1", hasS3: true, hasGlacier: false },
```

Comment:
What happened to this?

Reply:
It's a legacy endpoint (and still can be used since this is all for auto-complete)